index.html

```diff
@@ -3817,122 +3817,97 @@ <h3>Introduction</h3>
 					preview).</p>
 			</section>
 
-			<section id="app-toc-html">
+			<section id="app-toc-html" class="informative">
 				<h3>HTML Structure</h3>
 
-				<p>To optimize the machine processing of an HTML table of contents by user agents, authors SHOULD adhere
-					to the following structuring guidelines.</p>
+				<p>The machine-readable table of contents is defined within an [[html]] <a
+						href="https://www.w3.org/TR/html/sections.html#the-nav-element"><code>nav</code> element</a>. As
+					described in <a href="#wp-table-of-contents"></a>, this <code>nav</code> element has to be both the
+					first element in the document and identifiable by a <code>role</code> attribute with the value
+						<code>doc-toc</code>.</p>
 
-				<p class="note">The following content model identifies the elements that are processed by user agents
-					and need to be included; it does not disallow the inclusion of other elements or content. Such
-					unspecified content is ignored by user agents, as described in <a href="#app-toc-ua"></a>.</p>
+				<p>Although the content model of the <code>nav</code> element is not restricted, there are rules that
+					have to be followed in order for a user agent to be able to extract the hierarchy of links. These
```

Comment:
"have to be followed" may sound too normative-ish? what about rephrasing to something along the line of: "UA will only be able to extract a meaningful toc if the content follows the following set of best-practices"